giraffe/src/components/Axes.tsx

```diff
@@ -100,9 +100,19 @@ export const drawAxes = ({
   context.textAlign = 'end'
   context.textBaseline = 'middle'
   const yDomainWidth = yDomain[1] - yDomain[0]
+  if (yColumnLabels) {
+    let y
```

Comment:
I see why you added this, but it's not necessary here.

`let` and `const` are block scoped, which means they only live while the block they are declared in is in scope. This becomes relevant in `if/else` and `try/catch` statements:

```ts
if (someCondition) {
  const myVariable = 'true'
} else {
  const myVariable = 'false'
}
console.log(myVariable) // myVariable is not defined
```
`myVariable` won't exist at the `console.log` statement. So the solution is to do this:

```js
let myVariable
if (someCondition) {
  myVariable = 'true'
} else {
  myVariable = 'false'
}
console.log(myVariable) // hey it worked!
```

But this `if` block doesn't have any situation where something needs to use `y`  when it's out of scope. So all of this is to say that you can define and declare `y` on every iteration of the loop:

```ts
const y = yScale(i) + margins.top - height / (yColumnLabels.length * 2)
```

```suggestion
```